.pre-commit-config.yaml

```diff
@@ -0,0 +1,41 @@
+repos:
+  - repo: https://github.com/psf/black
+    rev: 21.7b0
+    hooks:
+      - id: black
+        stages: [manual]
+        args:
+          - --safe
+          - --quiet
+        files: ^((homeassistant|script|tests)/.+)?[^/]+\.py$
+  - repo: https://gitlab.com/pycqa/flake8
+    rev: 3.9.2
+    hooks:
+      - id: flake8
+        stages: [manual]
+        additional_dependencies:
+          - pycodestyle==2.7.0
```

Comment:
Please make sure that the tox run is using the same linting as pre-commit, by running pre-commit in tox.

Reply:
That is a very good idea, will do.

Reply:
I have to research this a bit more.
- seems flake8 do not run in CI (it comes with a number of “suggestions” when I run it local)
- black runs with —check, where in pre-commit it make changes to the code.